### PR TITLE
chore(vyper-verify): added vyper verify plugin doc with examples

### PR DESCRIPTION
**Pull request by kiriyaga**
Added vyper verify plugin user doc with examples of usage.

### CODE REVIEW

File: docs/tools/hardhat/hardhat-zksync-vyper-verify.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# `hardhat-zksync-vyper-verify`
+
+This plugin is used to verify vyper contracts on the zkSync Era network.
+
+[Changelog](https://github.com/matter-labs/hardhat-zksync/blob/main/packages/hardhat-zksync-vyper-verify/CHANGELOG.md)
```
Comment (uF4No) [inline docs/tools/hardhat/hardhat-zksync-vyper-verify.md:5]: This currently returns a 404. Will we have the Changelog with the relelase?

File: docs/tools/hardhat/hardhat-zksync-vyper-verify.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+```sh
+yarn hardhat vyper:verify --network <network> <contract address> --contract <fully qualified name>
+```
+
+With the `--contract` parameter you can also specify which contract from your local setup you want to verify by specifying its Fully qualified name. Fully qualified name structure looks like this: "contracts/Contract.vy:Contract" <br/>
+
+#### Constructor arguments
+
+If your contract was deployed with the specific constructor arguments, you need to specify them when running the verify task. For example:
```
Comment (uF4No) [inline docs/tools/hardhat/hardhat-zksync-vyper-verify.md:81]:
```suggestion
If your contract was deployed with specific constructor arguments, you need to specify them when running the verify task. For example:
```

File: docs/tools/hardhat/hardhat-zksync-vyper-verify.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+```sh
+yarn hardhat vyper:verify --network testnet 0x7cf08341524AAF292255F3ecD435f8EE1a910AbF "Hi there!"
+```
+
+If your constructor takes a complex argument list, you can write a separate javascript module to export it. For example, create an `arguments.js` file with the following structure:
```
Comment (uF4No) [inline docs/tools/hardhat/hardhat-zksync-vyper-verify.md:87]:
```suggestion
If your constructor takes a complex argument list, you can write a separate Javascript module to export it. For example, create an `arguments.js` file with the following structure:
```

File: docs/tools/hardhat/hardhat-zksync-vyper-verify.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+The following command checks the status of the verification request for the specific verification ID:
+
+```sh
+yarn hardhat verify-status:vyper --verification-id <your verification id>
+```
+
+### Verify smart contract programmatically
+
+If you need to run the verification task directly from your code, you can use the hardhat `verify:verify:vyper` task with the previously mentioned parameters with the difference in using `--address` parameter when specifying contarct's address.
```
Comment (uF4No) [inline docs/tools/hardhat/hardhat-zksync-vyper-verify.md:133]:
```suggestion
If you need to run the verification task directly from your code, you can use the hardhat `verify:verify:vyper` task with the previously mentioned parameters with the difference in using `--address` parameter when specifying contract's address.
```

Typo although I don't understand what this part means: " with the difference in using `--address` parameter when specifying contract's address."

File: docs/tools/hardhat/hardhat-zksync-vyper-verify.md (hunk 1 above)
Comment (kiriyaga) [reply docs/tools/hardhat/hardhat-zksync-vyper-verify.md:5]: Yes, that is correct.

File: docs/tools/hardhat/hardhat-zksync-vyper-verify.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+module.exports = "0x0x00087a676164696a61310000087a676164696a61310000000000000000000000008537b364a83f5c9a7ead381d3baf9cbb83769bf5";
+```
+
+### Verification status check
+
+The verification process consists of two steps:
+
+- A verification request is sent to confirm if the given parameters for your contract are correct.
+- Then, we check the verification status of that request.
+  Both steps run when you run the `vyper:verify` task, but you will be able to see your specific verification request ID.
```
Comment (idea404) [inline docs/tools/hardhat/hardhat-zksync-vyper-verify.md:122]:
```suggestion
  Both steps run when you run the `verify:vyper` task, but you will be able to see your specific verification request ID.
```

**Review by idea404 — CHANGES_REQUESTED** [review]
The current version pulled after installing from npm (`^0.0.1-alpha.1`) runs with the command `yarn hardhat verify:vyper` as opposed to what's been written in the doc. I've suggested changes to address this. Seems to work well for a simple greeter contract, will try out more complex examples.

**Pull request closed without merging**